ConsoleVirtualGUI.cs

```diff
@@ -0,0 +1,53 @@
+﻿
+namespace BetterProductInventory
+{
+    public class ConsoleVirtualGUI
+    {
+        
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Welcome to this Mangment System, please choose what you want to do!");
+
+            while (true)
+            {
+                Console.WriteLine("1.Add a New Product");
+                Console.WriteLine("2.View All Existing Products");
+                Console.WriteLine("3.Edit an Exisiting Product");
+                Console.WriteLine("4.Delete a Product");
+                Console.WriteLine("5.Search For a Product");
+                Console.WriteLine("6.Close");
+                int Choice = Convert.ToInt32(Console.ReadLine());
```

Comment:
you should have `Input Validation` 
You might want to handle this by using `int.TryParse` to validate and convert the input.